api/src/main/java/org/motechproject/nms/api/web/CallDetailsController.java

```diff
@@ -126,7 +126,7 @@ private void createCallDetailRecord(FrontLineWorker flw, CallDetailRecordRequest
             content.setEndTime(new DateTime(callContentRequest.getEndTime() * MILLISECONDS_PER_SECOND));
 
             if (service == Service.MOBILE_KUNJI) {
-                content.setMobileKunjiCardNumber(callContentRequest.getMkCardNumber());
+                content.setMobileKunjiCardNumber(callContentRequest.getMkCardCode());
```

Comment:
Probably.  Since it has a different prefix the refactor didn't catch it.

On Mon, May 11, 2015 at 2:32 PM, Lauren Lavoie <notifications@github.com>
wrote:

> In
> api/src/main/java/org/motechproject/nms/api/web/CallDetailsController.java
> <https://github.com/motech-implementations/mim/pull/234#discussion_r30084742>
> :
>
> > @@ -126,7 +126,7 @@ private void createCallDetailRecord(FrontLineWorker flw, CallDetailRecordRequest
> >              content.setEndTime(new DateTime(callContentRequest.getEndTime() * MILLISECONDS_PER_SECOND));
> >
> >              if (service == Service.MOBILE_KUNJI) {
> > -                content.setMobileKunjiCardNumber(callContentRequest.getMkCardNumber());
> > +                content.setMobileKunjiCardNumber(callContentRequest.getMkCardCode());
>
> nitpick: should we also rename CallContent.mobileKunjiCardNumber to
> CallContent.mobileKunjiCardCode?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/motech-implementations/mim/pull/234/files#r30084742>.
>